Example/GiniVision/ComponentAPIAnalysisViewController.swift

```diff
@@ -19,7 +19,7 @@ class ComponentAPIAnalysisViewController: UIViewController {
     /**
      The image data of the captured document to be reviewed.
      */
-    var imageData: Data!
+    var document: GiniVisionDocument!
```

Comment:
I guess the controller cannot work without a document. Would it be possible to not have it as an optional?

Reply:
The idea would be injecting it on the constructor, but since we are using `Storyboards` it is not possible to use its initializer, is it?

Reply:
It's not, but still, I don't think having it force unwrapped is so good. Can you at least use "?"?